src/methods/send_invoice.rs

```diff
@@ -55,6 +92,8 @@ pub struct SendInvoice<'a> {
     allow_sending_without_reply: bool,
     #[serde(skip_serializing_if = "Option::is_none")]
     reply_markup: Option<inline::Keyboard>,
+    #[serde(flatten, skip_serializing_if = "Option::is_none")]
```

Comment:
```suggestion
    #[serde(flatten)]
```
Works without `skip_serializing_if` anyway